Bento/Views/Box.swift

```diff
@@ -22,6 +22,8 @@ infix operator |---*: NodeConcatenationPrecedence
 infix operator |---?: NodeConcatenationPrecedence
 infix operator <>: ComposingPrecedence
 
+/// Box keeps array of section which should be displayed on the screen
+/// Box **doesn't have** a visual representation
```

Comment:
This comment should have a period `.`, or it will be a long sentence in Xcode tooltip.